packages/core-typings/src/IMessage/IMessage.ts

```diff
@@ -232,3 +232,5 @@ export type IMessageInbox = IMessage & {
 
 export const isIMessageInbox = (message: IMessage): message is IMessageInbox => 'email' in message;
 export const isVoipMessage = (message: IMessage): message is IVoipMessage => 'voipData' in message;
+
+export type IOmniRoomClosingMessage = Pick<IMessage, 't' | 'groupable'> & Partial<Pick<IMessage, 'msg'>>;
```

Comment:
looks wrong, this looks more like a `system message` than IOminichannelRoomClosingSomething
I think this is a internal type, not a core one

Reply:
Changes done :heavy_check_mark: